src/identity/apis/org.ts

```diff
@@ -34,6 +38,55 @@ export type QuartzOrganizations = {
   status?: RemoteDataState
 }
 
+// create a new organization
+export const createNewOrg = async (
+  organizationCreateRequest: OrganizationCreateRequest
+) => {
+  const {orgName, provider, region} = organizationCreateRequest
+
+  const response = await postOrg({
+    data: {
+      orgName,
+      provider,
+      region,
+    },
+  })
+
+  if (response.status === 401) {
+    throw new UnauthorizedError(response.data.message)
+  }
+
+  if (response.status === 409) {
+    throw new OrgNameConflictError(response.data.message)
+  }
+
+  if (response.status === 422) {
+    throw new UnprocessableEntityError(response.data.message)
+  }
+
+  if (response.status === 500) {
+    throw new ServerError(response.data.message)
+  }
+
+  // Status code in response when successfully creating an org is 201.
+  return response.data
+}
+
+// fetch the list of clusters in which an org can be created
+export const fetchClusterList = async () => {
```

Comment:
this method is called `fetchClusterList` but below we have `getDefaultAccountDefaultOrg`, but below that, we also have `fetchOrbsByAccountID`. can we make `fetch` and `get` verbiage consistent here? or point out what is different about them?

Reply:
I'll make an edit to change this to `fetch` throughout, which is probably clearer:
https://stackoverflow.com/questions/2141818/method-names-for-getting-data

> Example: [fetch](https://www.google.com/search?q=define%20fetch) implies that some entity needs to go and get something that is remote and bring it back. Dogs fetch a stick, and [retrieve](https://www.google.com/search?q=define%20get) is a synonym for fetch with the added semantic that you may have had possession of the thing prior as well. [get](https://www.google.com/search?q=define%20get) is a synonym for [obtain](https://www.google.com/search?q=define%20obtain) as well which implies that you have sole possession of something and no one else can acquire it simultaneously.